index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>McLaren</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  
+</head>
+
+  <body>
+  
+    <header>
+
+      <div class="headerLog">
+        <div class="headerLogFacebook"><i class="fa-brands fa-facebook-f"></i></div>
+        <div class="headerLogTwitter"><i class="fa-brands fa-twitter"></i></div>
+        <div class="headerLogLanguage"><a href="">English</a></div>
+        <div class="headerLogLanguage"><a href="">My Page</a></div>
+        <div class="headerLogout"><a href="">Logout</a></div>
+      </div>
+
+      <nav>
+        <div class="navLogo"><img src="images/mclarenlogo..png"></div>
+        <div class="nav"><a href="about.html">About</a></div>
+        <div class="nav"><a>Program</a></div>
+        <div class="nav"><a>Join</a></div>
+        <div class="nav"><a>Sponsor</a></div>
+        <div class="nav"><a>News</a></div>
+        <div class="navCampaign"><a>Campaign</a></div>
+      </nav>
```

Comment:
- [ ] Please you are required to stick to design while personalizing the contents. Kindly change the background color of your header to match the design guidelines.
- [ ] Also, kindly add the spaces at the sides of the contents of the header as seen in the design template.
 ### Your design
![1](https://user-images.githubusercontent.com/94551484/192071562-2d6c2bb9-a258-46b1-825b-d6dca0c1b5da.png)
### Design guidelines
![f1](https://user-images.githubusercontent.com/94551484/192071649-b324a615-e97e-4987-b4e1-eaae95403848.png)